### PR TITLE
Update rabbitmq.yml

### PR DESCRIPTION
**Pull request by KnechtionsCoding**
Adding in variable password per user. A list of items with unique passwords rather than a common password.

### CODE REVIEW

**Comment by KnechtionsCoding** [comment]
@webknjaz I'm not allowed to merge this, but I think it is still valuable :)

File: language_features/rabbitmq.yml [hunk 1]
```diff
@@ -26,10 +26,10 @@
     - restart rabbitmq
 
   - name: add users
-    rabbitmq_user: user={{item}} password=changeme tags=administrator,{{item}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
+    rabbitmq_user: user={{item.username}} password={{item.password}} tags=administrator,{{item}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
```
Comment (webknjaz) [inline language_features/rabbitmq.yml]: Doesn't new form of `{{item}}` break things?
```suggestion
    rabbitmq_user: user={{item.username}} password={{item.password}} tags=administrator,{{item.username}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
```

Comment (KnechtionsCoding) [reply language_features/rabbitmq.yml]: Honestly, I'm not sure. I've not touched this in two years. If it is supposed to be item.username/password I'll trust you and commit that suggestion

Comment (webknjaz) [reply language_features/rabbitmq.yml]: It seems logical to keep the original value there

**Pull request merged**